tv-shows-app/src/app/page.tsx

```diff
@@ -0,0 +1,9 @@
+'use client'
+
+import { ShowReviewSection } from "@/components/features/shows/ShowReviewSection/ShowReviewSection";
+
+export default function Home() {
+	return (
+		<ShowReviewSection />
+	);
```

Comment:
U ovakvom slučaju možeš ovo direktno vratiti:
```suggestion
	return <ShowReviewSection />;
```